cbmc/proofs/Makefile.common

```diff
@@ -117,7 +118,7 @@ report: cbmc.txt property.xml coverage.xml
 clean:
 	$(RM) $(OBJS) $(ENTRY).goto
 	$(RM) $(ENTRY)[0-9].goto $(ENTRY)[0-9].txt
-	$(RM) cbmc.txt property.xml coverage.xml TAGS
+	$(RM) cbmc.txt property.xml coverage.xml TAGS TAGS-*
```

Comment:
`s/ TAGS TAGS-* / TAGS* /g`

This is also used with `veryclean`, so if you want to add it here, remove it there.

Reply:
I've removed it from veryclean.  I'm generally scared to do things like TAGS* that could match so much when it is easy to be more specific about what I want to remove.